sdks/python/tests/unit/evaluation/metrics/test_heuristics.py

```diff
@@ -1,9 +1,11 @@
+import pytest
```

Comment:
don't forget to add `nltk` to `tests/unit/test_requirements.txt`, or these unit tests won't work in our CI